### PR TITLE
Fix: show center details without network

### LINKED ISSUE #752: Center details not visible when Internet connection is off
**Summary:** 

Complete details of centers which are synced with database is not shown when Internet connection is off.

**Steps to reproduce:** 

Go to Center List Activity and sync some centers. After that enable offline mode and disable internet connection. Then refresh list and click on any item.

**Expected behavior:** 

Details should be shown.

**Observed behavior:** 

Details are not shown and an error message "Failed to fetch center info" is shown.

**Screenshots:** 

centersyncyoyo

### PR DESCRIPTION
**Pull request by mayank-kgp**
Resolves #752 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@therajanmaurya Please have a look at it.

**Comment by mayank-kgp** [comment]
@therajanmaurya @puneetkohli521 Please see GIF.
centersyncyo

**Comment by therajanmaurya** [comment]
This PR is invalid, Please include the migration table code.

File: mifosng-android/src/main/java/com/mifos/api/local/database/MigrationVersion2.java [hunk 1]
```diff
@@ -56,4 +57,19 @@ public void onPreMigrate() {
             addColumn(SQLiteType.INTEGER, "centerDate_year");
         }
     }
+
+    @Migration(version = MifosDatabase.VERSION, database = MifosDatabase.class)
```
Comment (mayank-kgp) [inline mifosng-android/src/main/java/com/mifos/api/local/database/MigrationVersion2.java]: @therajanmaurya Migration code added.

File: mifosng-android/src/main/java/com/mifos/api/local/databasehelper/DatabaseHelperRunReport.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    /**
+     * This Method  write the Center Summary in tho DB. According to Schema Defined in Model
+     * @param centerInfos Model of List of Center Info
+     * @param centerId       Center Id
+     * @return List<CenterInfo>
+     */
+    public Observable<List<CenterInfo>> saveCenterSummaryInfo(final List<CenterInfo> centerInfos,
+                                              final int centerId) {
+        AsyncTask.THREAD_POOL_EXECUTOR.execute(new Runnable() {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/api/local/databasehelper/DatabaseHelperRunReport.java]: don't use `AsyncTask.THREAD_POOL_EXECUTOR` use `Observable.defer()`

**Comment by mayank-kgp** [comment]
@therajanmaurya Changes made.

**Comment by mayank-kgp** [comment]
@therajanmaurya Please review. :)

**Comment by tarun0** [comment]
@mayank-kgp Can you fix the conflicts? Please reply within a week if you can.

**Comment by tarun0** [comment]
Closing for lack of updates.

**Pull request closed without merging**